src/prefect/testing/cli.py

```diff
@@ -34,7 +35,7 @@ def check_contains(cli_result: Result, content: str, should_contain: bool):
     if should_contain:
         assert (
             content in output
-        ), f"Desired contents {display_content!r} not found in CLI ouput"
+        ), f"Desired contents {display_content!r} not found in CLI ouput\n{output!r}"
```

Comment:
`invoke_and_assert` already displays all of the output by default on L75/84

Reply:
ah sorry, i wasn't seeing that for some reason - i can remove

Reply:
@madkinsz updated